### PR TITLE
Refactor: API key page design details

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

Adds small enhancements to the API key page:
- Expiration input will start with a predefined date pre-selected if applicable
- Expiration date alerts when key is expiring/expired
- Helper text for expiration date when using a predefined date option
- Fix the word "scopes" always being in the plural form

## Test Plan

Manual

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by TorstenDittmann** [comment]
When trying to create an API key 😆 
Bildschirm­foto 2023-02-10 um 11 37 39

**Comment by TGlide** [comment]
> When trying to create an API key 😆 Bildschirm­foto 2023-02-10 um 11 37 39

Oops 🤣 Fixed!

File: src/routes/console/project-[project]/overview/keys/[key]/updateExpirationDate.svelte [hunk 1]
```diff
@@ -1,12 +1,14 @@
 <script lang="ts">
     import { invalidate } from '$app/navigation';
     import { trackEvent } from '$lib/actions/analytics';
+    import Alert from '$lib/components/alert.svelte';
     import CardGrid from '$lib/components/cardGrid.svelte';
     import Heading from '$lib/components/heading.svelte';
     import { Dependencies } from '$lib/constants';
     import { Form } from '$lib/elements/forms';
     import Button from '$lib/elements/forms/button.svelte';
     import FormList from '$lib/elements/forms/formList.svelte';
```
Comment (TorstenDittmann) [inline src/routes/console/project-[project]/overview/keys/[key]/updateExpirationDate.svelte:10]: mind cleaning up these imports to named ones? 🙂

**Review by TorstenDittmann — APPROVED** [review]

File: src/routes/console/project-[project]/overview/keys/[key]/updateExpirationDate.svelte (hunk 1 above)
Comment (TGlide) [reply src/routes/console/project-[project]/overview/keys/[key]/updateExpirationDate.svelte]: Done 😉

**Pull request merged**